x/ratelimit/keeper/rate_limit.go

```diff
@@ -151,41 +187,95 @@ func (k Keeper) GetAllRateLimits(ctx sdk.Context) []types.RateLimit {
 	return allRateLimits
 }
 
+// Sets the sequence number of a packet that was just sent
+func (k Keeper) SetPendingSendPacket(ctx sdk.Context, channelId string, sequence uint64) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.PendingSendPacketPrefix)
+	key := types.GetPendingSendPacketKey(channelId, sequence)
+	store.Set(key, []byte{1})
```

Comment:
What's the `[]byte{1}` needed for?

Reply:
All we really need to store is the key, the value doesn't matter as long as it's non-empty.

The implementation can be thought of more as a a set in python, rather than a map